### DISCUSSION TITLE
Please allow for a bigger batch count (number of images to generate =+1000)

### DISCUSSION THREAD

**Discussion by dmt-hub** [hlky/stable-diffusion-webui#515] [Feature suggestions]
How can i set a bigger batch count than 50 in the webui? Can i change this myself somewhere?
Please allow for a batch count of 1000 or more.. 

Thank you.

**Comment by rcyrus** [comment]
you can change the limit here: https://github.com/hlky/stable-diffusion-webui/blob/320ea01d113096e5d99715b3adbe6b82eb9c0059/frontend/frontend.py#L34

Be aware that your change will get clobbered next time you update your local copy.

**Reply by dmt-hub** [reply]
Thanks alot!

**Comment by JoshuaKimsey** [comment]
This could potentially be made into a parser `--argument` for setting this at launch? I'm not sure actually, but if it could that would allow people to set their own (potentially ridiculous) values if they have a need for it, not just for batch count either.  @altryne you likely know more about this than I do. Can a parser argument be passed at launch to allow people to set custom ranges for the sliders? Or because those values are set in `frontend.py`, does that mean a parser argument passed to `webui.py` wouldn't be able to easily change those values?

**Reply by altryne** [reply]
We can def allow "maximum values" but getting that setting per slider will add... what? 50 settings? 🤯 
Arbitrary values are possible with dynamic parameters by just typing `number:1000` in the input box

**Reply by JoshuaKimsey** [reply]
Ah, that's true! I was thinking it could be an easy solution to the issue of sliders with overly large ranges, but you're right the input box is likely a much easier way to work with that. Might wanna add some bit of info on that page that says that can be done? Maybe a small pop-out window that gives some facts of the input box abilities?

**Comment by altryne** [comment]
> How can i set a bigger batch count than 50 in the webui? Can i change this myself somewhere? Please allow for a batch count of 1000 or more..
> 
> Thank you.

Web ui will and be mostly unresponsive. The gallery is not built for more than 20-40 pictures or so
You can use the CLI mode of this way of usage btw

**Comment by Vyviel** [comment]
Could just add a run forever option and you manually stop it with the stop button?

**Reply by altryne** [reply]
Stop button could be cool. we currently don't have one and there are 2 PRs on their way with a try for one

**Comment by Wang-Thunder** [comment]
Use the dynamic tags. Add -n 1000 to your prompt and it will increase the batch size.
